src/components/RightHandMenu/DonutChart/Legend/Legend.css

```diff
@@ -0,0 +1,29 @@
+.leg__item {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 4px 8px;
+}
+
+.leg__left {
+  display: flex;
+  align-items: center;
+}
+
+.leg__color {
+  width: 15px;
+  height: 15px;
+  border-radius: 3px;
+  margin-right: 15px;
+}
+
```

Comment:
The legend is bleeding over the container on the bottom for me - can we make sure it fits into its container? (I'm fine if this is a separate PR)

Reply:
Fixed.... Nice catch 👍👍👍